packages/user-flow-cli-testing/src/lib/user-flow-cli.ts

```diff
@@ -73,7 +72,7 @@ export class UserFlowCliProject extends CliProject<RcJson> {
     await kill({ port: this.serveCommandPort });
   }
 
-  $init(processParams?: Partial<InitCommandArgv & GlobalOptionsArgv>, userInput?: string[]): Promise<TestResult> {
+  $init(processParams?: Partial<{} & GlobalOptionsArgv>, userInput?: string[]): Promise<TestResult> {
```

Comment:
The type was originally incorrect. Now its worse. 

Will fix before ready to merge